packages/iptables/data_stream/log/_dev/test/system/test-journald-config.yml

```
@@ -1,4 +1,7 @@
 service: iptables-log-journald
+skip:
+  reason: "Panics observed while using the journald input"
```

It looks good, but I believe we could make it explicit the issue is not on our code. Maybe something like that: 
```suggestion
  reason: "A bug on the host journald causes our journald input to panic"
```

---

Updated, thanks!